test/lib/jdk/test/lib/security/SecurityUtils.java

```diff
@@ -125,5 +127,33 @@ private static boolean anyMatch(String value, List<String> algs) {
         return false;
     }
 
+    public static void inspectTlsBuffer(ByteBuffer buffer) throws IOException {
```

Comment:
Another take/leave minor nit:  maybe `dumpTlsPacketsBuffer` as that's all you're doing.

Reply:
I think `dump` indicates that we dump the content of the buffer, that's not what we are doing, we are inspecting the buffer.